packages/askui-nodejs/src/execution/client-interface.ts

```diff
@@ -2,9 +2,14 @@ import { CredentialArgs } from '../utils/http/credentials';
 import { AnnotationLevel } from './annotation-level';
 
 export interface ClientArgs {
+  // The adress of the askui UI Controller server.
   readonly controlServerUrl?: string,
+  // Address of the askui Inference server.
   readonly controlYourUiApi?: string,
+  /* Usage of annotate command after execution of test steps.
+  You have three options: `DISABLED`, `ON_FAILURE`, `ALL`. */
   readonly annotationLevel?: AnnotationLevel,
+  // Credentials for http client.
```

Comment:
I think you can and should be more specific as this only referes to the `controlYourUiApi` (⟶ should be renamed).